pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -289,6 +289,27 @@ def test_setitem_periodindex(self):
         assert isinstance(rs.index, PeriodIndex)
         tm.assert_index_equal(rs.index, rng)
 
+    @pytest.mark.parametrize("klass", [list, np.array])
+    def test_iloc_setitem_bool_indexer(self, klass):
```

Comment:
test name is good, belongs in tests.indexing.test_iloc

Reply:
Done